internal/docs/pubsub/design.md

```diff
@@ -464,10 +413,10 @@ func (t *Topic) Close() error {
 	return t.driver.Close()
 }
 
-// NewTopic makes a pubsub.Topic from a driver.Topic and opts to
-// tune how messages are sent. Behind the scenes, NewTopic spins up a goroutine
-// to bundle messages into batches and send them to the server.
-func NewTopic(d driver.Topic, opts TopicOptions) *Topic {
+// NewTopic makes a pubsub.Topic from a driver.Topic. Behind the scenes,
+// NewTopic spins up a goroutine to bundle messages into batches and send
```

Comment:
This is a similar implementation detail that should go from the function API documentation and into the function body.

Reply:
Done.